adoption/src/main/resources/standards/adoption/messages/adoption-1.0.0-example.json

```diff
@@ -0,0 +1,60 @@
+{
+  "lastUpdatedDateTime": "2024-06-18T20:44:37+01:00",
+  "interval": "day",
+  "startDate": "2024-06-16",
+  "endDate": "2024-06-22",
```

Comment:
Is this correct? It looks like the original example that you questioned when we talked about how to do this scenario.

Reply:
You're right, it is not correct indeed. George and me talked about the standard this afternoon and he is going to update the spec to (or ask Henrik to do so).